server/src/main/scala/net/wiringbits/models/UserToken.scala

```diff
@@ -0,0 +1,15 @@
+package net.wiringbits.models
+
+import java.util.UUID
+
+case class UserToken(userId: UUID)
```

Comment:
Don't we need the token too?

Reply:
Yes, indeed, but you told me to split the PR in three. Right now the email endpoint only consists in the user id (very dangerous). I was planning of include the token approach after this PR gets merged

Reply:
Thanks for clarifying